ios/MattermostShare/SessionManager.m

```diff
@@ -196,6 +201,9 @@ -(void)sendPostRequestForId:(NSString *)requestWithGroup {
     NSURLSessionDataTask *createTask = [createSession dataTaskWithRequest:request];
     NSLog(@"Executing post request");
     [createTask resume];
+    self.closeExtension();
```

Comment:
I'm not familiar enough with the domain to answer this, but I assume this is the only place where `closeExtension` ought to be called? That is, this is the only exit path of relevance?

Reply:
Yes this is the only place that is being called, but in case of an error the extension is being closed from JS code. also the reason I did it this way is cause I wanted to avoid passing the extensionContext as an argument